lampod/src/ln/inventory_manager.rs

```diff
@@ -41,12 +41,21 @@ impl InventoryHandler for LampoInventoryManager {
                     .map(|alias| alias.to_string());
                 // we have to put "" in case of alias missing as cln provide us with a random alias.
                 let alias = alias.unwrap_or_else(|| "".to_string());
+                let (_, height) = self.channel_manager.onchain.backend.get_best_block()?;
+                let blockheight;
+                if height.is_none() {
+                    blockheight = "Still loading...".to_string();
```

Comment:
if it is still loading you should emit a warning

Reply:
But I have added the warning field in ec08e9f7edeee335a041f350117c2f11f3e618fb

Reply:
it is wrong :) sorry we can discuss just the feature inside this PR? 

The warnings do not need to be related to the bitcoin core, this is another problem